apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-info.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Box,
+  Flex,
+  Grid,
+  IconButton,
+  Text,
+  Separator,
+  Button,
+} from "@webstudio-is/design-system";
+import type { Asset } from "@webstudio-is/prisma-client";
+import { getFormattedAspectRatio } from "./utils";
+import {
+  CloudIcon,
+  AspectRatioIcon,
+  Cross1Icon,
+  SizeIcon,
+  TrashIcon,
+} from "@webstudio-is/icons";
+import { useSubmit } from "@remix-run/react";
+import prettyBytes from "pretty-bytes";
+
+const truncatedText = {
+  position: "relative",
+  maxWidth: 80,
```

Comment:
there is no way to avoid hardcoding this? its better than chars but still is gonna break if we change layout, right?

Reply:
alternative may be to use the fixed box width we already have and calculate from that, so at least if it changes there, it changes here as well, unless we can use layout somehow to enforce this

Reply:
The tooltip already has a width of 240

Reply:
you saying we can't be not hardcoding the max width?

Reply:
I don't know, from what I tried no, that was my first attempt

Reply:
I was wrong, I tried again and it worked, we now use 70%